src/interface/character/Parses.js

```diff
@@ -6,7 +6,7 @@ import { withRouter } from 'react-router-dom';
 import { compose } from 'redux';
```

Comment:
This file has a lot of stylistic changes that deviate from the style in the project. Please revert these style changes.